app/models/foreman_virt_who_configure/config.rb

```diff
@@ -8,7 +8,7 @@ class Config < ActiveRecord::Base
       # We keep both params permitted for compatibility with 1.11
       :listing_mode, :filtering_mode
     ]
-    audited :except => [ :hypervisor_password, :last_report_at, :out_of_date_at ]
+    audited :except => [ :hypervisor_password, :last_report_at, :out_of_date_at ], :associations => []
```

Comment:
this is unrelated but workarounds the audit association definition issue in dev env until audited supports associations natively (problem since Foreman 1.18)

Reply:
decided to open separate PR - https://github.com/theforeman/foreman_virt_who_configure/pull/88